.gitignore

```diff
@@ -1,4 +1,5 @@
 bin
+web
```

Comment:
how does this get generated? can we document everything we're gitignoring?

Reply:
Gonna rewrite the main readme with a lot of important info